lib/DerbyViewPlugin.js

```diff
@@ -24,6 +24,8 @@ DerbyViewsPlugin.prototype.apply = function(compiler) {
       virtualViewFilesInitialized = true;
       Object.entries(appNameToPath).forEach(([appName, requirePath]) => {
         const appPath = require.resolve(requirePath, { paths: [rootPath] });
+        const app = require(appPath);
+        viewCache.registerApp(app);
```

Comment:
In dev, this will cause the app to get re-registered. Which might be ok as long as it's the same instance in-memory.

Still, one way to be safe is to guard this with `if (!process.env.DERBY_HMR)`

Reply:
yeah just checked that and see duplicated build things happening

Reply:
☝️  Does not seem to be causing the additional build passes